modules/rosa-nodes-machine-pools-local-zones.adoc

```diff
@@ -0,0 +1,73 @@
+
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-nodes-machinepools-configuring.adoc
+:_content-type: PROCEDURE
```

Comment:
```suggestion
// * assemblies/rosa-nodes-machinepools-configuring.adoc

:_content-type: PROCEDURE
```

Just doing what everyone else does here. Not sure if this spacing matters, though.